### PR TITLE
[homekit] Update to iOS 10.3 beta 1

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

**Comment by VincentDondain** [comment]
Known `monotouch-test` bug: https://bugzilla.xamarin.com/show_bug.cgi?id=51801

**Comment by VincentDondain** [comment]
Known `link-all` bug: https://bugzilla.xamarin.com/show_bug.cgi?id=46097

File: src/HomeKit/HMEnums.cs [hunk 1]
```diff
@@ -411,6 +411,14 @@ public enum HMCharacteristicType {
...
 		ImageMirroring,
 
+		[iOS (10,3), Watch (3,2), TV (10,2)]
+		[Field ("HMCharacteristicTypeLabelNamespace")]
+		LabelNamespace,
+
+		[iOS (10,3), Watch (3,2), TV (10,2)]
+		[Field ("HMCharacteristicTypeLabelIndex")]
+		LabelIndex,
+
```
Comment (spouliot) [inline src/HomeKit/HMEnums.cs]: we missed it but that's incorrect - you cannot add enum values (without numbering them) in the middle of others without breaking all the one that comes after, see API diff results

Comment (VincentDondain) [inline src/HomeKit/HMEnums.cs]: Roger that! 👍

**Pull request merged**